core/display.js

```diff
@@ -896,16 +896,17 @@ export default class Display {
                             rect.img.src = rect.src;
                             rect.type = 'img';
                             break;
-                        case 'transparent':
+                        case 'transparent': {
                             let imageBmpPromise = createImageBitmap(rect.arr);
-                            imageBmpPromise.then( function(img) {
+                            imageBmpPromise.then((img) => {
                                 this._transparentOverlayImg = img;
                                 if (!this.enableCanvasBuffer) {
                                     this._enableCanvasBuffer = true;
                                 }
-                            }.bind(this) );
+                            });
                             this._transparentOverlayRect = rect;
                             break;
+                        }
```

Comment:
If you are going to bracket case statements you would need to be consistent and do it everywhere. Even within this same switch, the first case statement does not use brackets but this one now does. I don't necessarily care whether brackets are used or not, but it should be consistent. I wouldn't want to refactor the entire code base, so I would lean towards reverting the brackets here.